workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -2,22 +2,22 @@
 
 namespace Improbable.Gdk.Mobile.Android
 {
+    public enum AndroidDeviceType
+    {
+        Unknown,
+        Physical,
+        Emulator
+    }
+
     public static class AndroidDeviceInfo
     {
         public const string AndroidStudioEmulatorDefaultCallbackIp = "10.0.2.2";
 
-        private enum DeviceType
-        {
-            Unknown,
-            Physical,
-            Emulator
-        }
-
-        private static DeviceType activeDeviceType = DeviceType.Unknown;
+        private static AndroidDeviceType activeDeviceType = AndroidDeviceType.Unknown;
 
-        public static bool IsEmulator()
+        public static AndroidDeviceType ActiveDeviceType()
```

Comment:
we could also make this a property instead of a method

Reply:
👍

Reply:
it would look something like that:
```
private static AndroidDeviceType activeDeviceType = AndroidDeviceType.Unknown;
public static AndroidDeviceType ActiveDeviceType
{
    get
    {
        if (activeDeviceType == AndroidDeviceType.Unknown) 
        {
            // do the initialization
        }
        return activeDeviceType;
    }
}